tests/test_job.py

```diff
@@ -79,19 +79,32 @@ def test_custom_name(hq_env: HqEnv, tmp_path):
     assert len(table) == 2
 
 
-def test_custom_working_dir(hq_env: HqEnv, tmp_path):
-    offset = 10
+def test_custom_working_dir(hq_env: HqEnv, tmpdir):
     hq_env.start_server()
-    cwd_custom_tbl = hq_env.command(
+
+    cwd_offset = 9
+    test_string = "cwd_test_string"
+    test_file = tmpdir.mkdir("test_dir").join("testfile")
+    test_file.write(test_string)
+    test_file_path = str(tmpdir) + "/test_dir/"
+
+    cwd_submit_tbl = hq_env.command(
         [
             "submit",
-            "sleep 1",
-            "--cwd=/test/directory/",
+            "--cwd=" + test_file_path,
+            "--",
+            "bash",
```

Comment:
I suppose that using `[..., "cat", "testfile"]` should also work, but that's a detail :)